.ddev/.gitignore

```diff
@@ -1,13 +1,28 @@
 #ddev-generated: Automatically generated ddev .gitignore.
```

Comment:
It's not recommended to add this file to the git repo because it's managed by ddev

Reply:
got removed with latest commit